translator-files/po/de/Installation-for-Macintosh.po

```diff
@@ -43,11 +45,11 @@ msgstr "Installation auf macOS"
 #. type: Plain text
 #: wiki/en/Installation-for-Macintosh.md:9
 msgid ""
-"{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting Started"
-"\" branch1-url=\"Getting-Started\" %}"
+"{% include breadcrumb.html root=\"Using Jamulus\" branch1=\"Getting "
+"Started\" branch1-url=\"Getting-Started\" %}"
 msgstr ""
-"{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste Schritte"
-"\" branch1-url=\"Getting-Started\" %}"
+"{% include breadcrumb.html root=\"Jamulus Nutzen\" branch1=\"Erste "
+"Schritte\" branch1-url=\"Getting-Started\" %}"
```

Comment:
Not quite sure why Qt creator did that...

Reply:
What's happened here (and in lots of other places) is that different editors wrap the text at different points. It doesn't have any effect on the end result, though it does add noise to commits. There's not much that can be done other than ignore it.